### PR TITLE
chart: Update chart for the backing image feature

### PR DESCRIPTION
**Pull request by shuo-wu**
Longhorn #2006

### CODE REVIEW

File: chart/questions.yml [hunk 1]
```diff
@@ -436,6 +436,13 @@ Warning: This option works only when there is a failed replica in the volume. An
     type: int
     min: 0
     default: 0
+  - variable: defaultSettings.backingImageCleanupWaitInterval
+    label: Backing Image Cleanup Wait Interval
+    description: "This interval in minutes determines how long Longhorn will wait before cleaning up the backing image file when there is no replica in the disk using it."
```
Comment (joshimoo) [inline chart/questions.yml:441]: LGTM, just a question does 0 mean immediately or never?

**Review by joshimoo — APPROVED** [review]
LGTM

File: chart/questions.yml (hunk 1 above)
Comment (shuo-wu) [reply chart/questions.yml:441]: `0` means immediately.

**Comment by shuo-wu** [comment]
Can we merge this? @joshimoo

**Pull request merged**